src/core/public/rendering/_base.scss

```diff
@@ -73,7 +73,10 @@
   display: flex;
   flex-flow: column nowrap;
   margin: 0 auto;
-  min-height: calc(100vh - #{$euiHeaderHeightCompensation});
+  // IE needs this to be "height" instead of "min-height"
+  // "min-height" causes a bug described in the next link
+  // https://github.com/philipwalton/flexbugs#3-min-height-on-a-flex-container-wont-apply-to-its-flex-items
+  height: calc(100vh - #{$euiHeaderHeightCompensation});
```

Comment:
The problem with this switch is that the scrolls now must be managed by internal views. Not a huge deal, but we really try to keep the window scroll on the window instead of an individual element. If this problem truly only affected IE, let's keep `min-height` by default but change to `height` only for IE:

```scss
  min-height: calc(100vh - #{$euiHeaderHeightCompensation});

  @include internetExplorerOnly {
    // IE needs this to be "height" instead of "min-height"
    // "min-height" causes a bug described in the next link
    // https://github.com/philipwalton/flexbugs#3-min-height-on-a-flex-container-wont-apply-to-its-flex-items
    height: calc(100vh - #{$euiHeaderHeightCompensation});
  }
```

Reply:
@cchaos thanks for useful suggestion!
Done.